core/controllers/dashboard_test.py

```diff
@@ -463,6 +463,117 @@ def test_no_explorations(self):
         self.assertEqual(response['explorations_list'], [])
         self.logout()
 
+    def test_view_dashboard_no_explorations(self):
+        self.login(self.OWNER_EMAIL)
+        response = self.get_json(feconf.DASHBOARD_DATA_URL)
+        self.assertEqual(response['explorations_list'],[])
+
+        #testing that creator only visit dashboard without any exploration created
+        response = self.get_json(feconf.DASHBOARD_DATA_URL)
+        self.assertEqual(len(response['explorations_list']),0)
+        self.logout()
+
+        #Now viewing the dashboard without creating the exploration
+        self.login(self.VIEWER_EMAIL)
+        response = self.get_json(feconf.DASHBOARD_DATA_URL)
+        self.assertEqual(response['explorations_list'],[])
+        self.logout()
+
+    def test_single_exploration_creation_than_view_dashboard(self):
+        self.save_new_default_exploration(
+            self.EXP_ID, self.owner_id, title=self.EXP_TITLE)
+        rights_manager.assign_role_for_exploration(
```

Comment:
I was thinking about the editing and viewing limitations of dashboard that's why i have added it in the code but as you said it is unneccessary i will update the code again but can you please tell me where i am thinking wrong in the workflow ??